iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/pipeconsensus/PipeConsensusAsyncConnector.java

```diff
@@ -118,10 +115,18 @@ public void customize(PipeParameters parameters, PipeConnectorRuntimeConfigurati
     }
     consensusGroupId = parameters.getInt(PipeConnectorConstant.CONNECTOR_CONSENSUS_GROUP_ID_KEY);
 
+    // Get consensusPipeName from parameters passed by PipeConsensusImpl
+    if (!parameters.hasAttribute(PipeConnectorConstant.CONNECTOR_CONSENSUS_PIPE_NAME)) {
+      throw new PipeException(CUSTOMIZE_EXCEPTION_MSG);
```

Comment:
Better use "validate" to keep the semantics.

Reply:
good catch, fixed